Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -160,11 +163,19 @@ private void VerifyWebApplicationStarted (Uri webApplicationRoot, TimeSpan appli
         if (remainingTimeout <= 0)
         {
           s_log.Warn($"Checking the web application root '{webApplicationRoot}' timed out (HTTP status code: '{statusCode}').");
+
+          // Embed the response text as inner exception
+          var innerException = lastResponseText != null
+              ? new Exception($"Last response text:{Environment.NewLine}{lastResponseText}")
+              : null;
+
           throw new WebException(
               $"Checking the web application root '{webApplicationRoot}' did not return '{HttpStatusCode.OK}' "
               + $"in the defined {nameof(applicationPingTimeout)} ({applicationPingTimeout}). "
               + $"Last received HTTP status code was: '{statusCode}'.",
-              WebExceptionStatus.Timeout);
+              innerException,
+              WebExceptionStatus.Timeout,
+              lastResponse);
```

Comment:
Do we have a chance of testing this before we merge it? I would like to see the look and feel of the output.